src/components/cards/vaccines-per-day-card/vaccines-per-day-card.jsx

```diff
@@ -158,10 +167,11 @@ export class VaccinesPerDayCard extends React.PureComponent {
           dates={dates}
           listPfizer={records.pfizer}
           listModerna={records.moderna}
+          listAstraZeneca={records.astraZeneca}
         />
         <p>
-          În cazul vaccinelor Pfizer BioNTech și Moderna sunt necesare două doze pentru
-          imunizare.
+          În cazul vaccinelor Pfizer BioNTech și Moderna sunt necesare două
```

Comment:
turns out i was wrong

the correct text is 
`În cazul vaccinelor Pfizer BioNTech, Moderna și AstraZeneca sunt necesare două doze pentru imunizare.`